torch/_dynamo/package.py

```diff
@@ -324,6 +341,65 @@ def install(self, backends: dict[_BackendId, Any]) -> None:
                     SerializedCode.to_code_object(guarded_code.dynamo_code),
                 )
 
-    def save(self) -> _DynamoCacheEntry:
+    def cache_entry(self) -> _DynamoCacheEntry:
         self.validate()
         return _DynamoCacheEntry(codes=list(self._codes.values()))
+
+
+@CacheArtifactFactory.register
+class EagerCacheArtifact(PrecompileCacheArtifact[Any]):
+    @staticmethod
+    def type() -> str:
+        return "precompile_eager"
+
+    def after_deserialization(self) -> Any:
+        return pickle.loads(self.content)
+
+
+class DynamoStore:
```

Comment:
nit: some doc here on the purpose of DynamoStore (handling disk level things + serialization etc)